README.md

```diff
@@ -1,53 +1,24 @@
-# Class Extensions and Inheritance
+# Class Extensions and Inheritance: `super`
 
 ## Learning Goals
 
-- Use the `extends` keyword
 - Use the `super` method
-- Explain when `class` inheritance should be used in JavaScript
 
 ## Introduction
 
-In JavaScript, as in other Object Oriented languages, we've learned
-that you can create `class`es and build methods that can perform
-actions on instance data, or specific to the `class`. What if you have
-`class`es that exhibit many of the same behaviors, such as `Cat`, `Dog`,
-and `Bird`, which all have a method for `speak`? In JavaScript, you can
-create "child" object `class`es that inherit methods from their "parent"
-`class`es. In this lesson, we'll discuss 2 ways of _extending_ functionality
-to other `class`es.
-
-## Use the `extend` Keyword
-
-To get started with inheriting `class` functionality, we utilize the `extends`
-keyword. `extends` is used in `class` declarations to create a `class` which
-is a _child_ of another `class`. 
-
-```js
-class Pet {
-  constructor(name) {
-    this.name = name;
-  }
-}
-
-// Inherits from Pet
-class Dog extends Pet {
-  speak() {
-    return `${this.name} says woof!`
-  }
-}
-
-let dog = new Dog("Shadow");
-
-dog.speak(); // Shadow says woof!
-```
+In addition to extending classes, we can build methods that perform
```

Comment:
Since this references the last example, I'd include a code snippet of it here. Only because it will keep students from having to flip back and forth if they forgot the example or stopped for a while between these two lessons

Reply:
Makes sense